### DISCUSSION TITLE
v1.1.9

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#340] [Announcements/公告]
**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/v1.1.8...v1.1.9

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.1.9'>v1.1.9</a>.</em>

**Comment by CN-xdd** [comment]
没有改动任何源码，直接用的您的 打包的时候提示这样 请问怎么解决  
异常：找不到signingKey/ANDROID_SIGNING_KEY。检查您在工作流程中的输入。
signingKey/ANDROID_SIGNING_KEY 这个在哪  怎么绑定  
请求指点下 谢谢  三天了   很烦

**Reply by acooler15** [reply]
是使用github action打包吗？

https://github.com/constanline/XQuickEnergy/blob/master/.github/workflows/release.yml#L30-L39

**Reply by CN-xdd** [reply]
是的  注册KEY解决了  但打包提示不能从服务器获取资源

**Reply by Zhangfanyu1222** [reply]
把缓存删了重新运行工作流试试

**Comment by cxqd888** [comment]
Toast加载成功的提示好像没有了

**Comment by heavenbeike** [comment]
送好友卡片任务，是送给谁？送什么卡片？送了还能要回来吗？送完能自动领奖励并抽取新卡片吗？

**Comment by z1028kk** [comment]
打赏功能关闭掉了吗 好像没有找到打赏界面

**Reply by Zhangfanyu1222** [reply]
这个是在作者仓库里的
[app/src/main/res/drawable/donation.jpg](https://github.com/constanline/XQuickEnergy/blob/master/app/src/main/res/drawable/donation.jpg?raw=true)
